tools/c7n_mailer/c7n_mailer/email_delivery.py

```diff
@@ -262,29 +262,44 @@ def send_smtp_email(self, smtp_server, message, to_addrs):
         smtp_connection.sendmail(message['From'], to_addrs, message.as_string())
         smtp_connection.quit()
 
+    def get_mimetext_with_headers(self, message, subject, from_addr, to_addrs, cc_addrs, priority):
```

Comment:
little unclear how this interacting with html messages?

Reply:
originally had this named as set_mimetext_headers but thought that was unclear as it was returning back a mimetext message, I can switch it back